### PR TITLE
Created Adri

### PR DESCRIPTION
**Pull request by adripo**
Thanks for this project!

### CODE REVIEW

File: domains/adri.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "adripo",
+    "email": "26493496+adripo@users.noreply.github.com"
```
Comment (TheBotlyNoob) [inline domains/adri.json:4]: this should be an email we can email you with.

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

File: domains/adri.json (hunk 1 above)
Comment (adripo) [reply domains/adri.json]: I changed it

**Comment by adripo** [comment]
@TheBotlyNoob I changed the email as requested

**Comment by adripo** [comment]
@TheBotlyNoob can you please approve my pr?

**Review by phenax — APPROVED** [review]
Sorry for the delay (#1529)

**Pull request merged**